docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -18,48 +18,44 @@ OpenMS User Tutorial
 ### Installation
 
 Before we get started, we will install OpenMS and KNIME. If you take part in a training session you will have likely received an USB stick from us that contains the required data and software. If we provide laptops with the software you may of course skip the installation process and continue reading the next section.
+If you are doing this tutorial online, choose online in the following tab(s).
 
-#### Installation from the OpenMS USB stick
+::::{tab-set}
+
+:::{tab-item} Online
+:sync: online
+
+If you are working through this tutorial at home/online, proceed with the following steps:
+
+- Download and install OpenMS using the installation instructions for your operating system:
+  - [GNU/Linux](/installations/installation-on-gnu-linux.md)
+  - [macOS](/installations/installation-on-macos.md)
+  - [Windows](/installations/installation-on-windows.md)
+- Download and install [KNIME](https://www.knime.org/downloads/overview)
+
+:::
+
+:::{tab-item} USB Stick
```

Comment:
:::{tab-item} USB Stick
:sync: usb
 
Please choose the directory that matches your operating system and execute the installer.

For Windows, you run:

The OpenMS installer: {{ '{path}'+'Windows,OpenMS-{0}-Win64.exe'.format(version) }}
The KNIME installer: {{ '{path}'+'Windows,KNIME-{0}-Installer-64bit.exe'.format(knime_version) }}
On macOS, you run:

The OpenMS installer: {{ '{path}'+'Mac,OpenMS-{0}-macOS.dmg'.format(version) }}
The KNIME installer: {{ '{path}'+'Mac,knime_{0}.app.macosx.cocoa.x86_64.dmg'.format(version) }}
On Linux, you can extract KNIME to a folder of your choice and for TOPPView you need to install OpenMS via your package manager or build it on your own with our [build instructions](https://github.com/OpenMS/OpenMS-docs/blob/825c456f8b0e4b25a10a8d45dd10e74761a04a48/docs/tutorials-and-quickstart-guides/installations/installation-on-gnu-linux.md#build-openms-from-source).

:::

::::